rootfs/scheduler/__init__.py

```diff
@@ -384,13 +432,17 @@ def run(self, name, image, entrypoint, command):
             name, image, entrypoint, command))
         appname = name.split('_')[0]
         name = name.replace('.', '-').replace('_', '-')
+        imgurl = self.registry + '/' + image
+        POD = POD_TEMPLATE
+        if image.startswith('http://') or image.startswith('https://'):
```

Comment:
RC has build type availability. Where as run doesn't know what buildtype it is unless we pass .